tests/app/celery/test_nightly_tasks.py

```diff
@@ -288,13 +289,13 @@ def test_delete_notifications_task_calls_task_for_services_that_have_sent_notifi
     nothing_to_delete_email_template = create_template(service_nothing_to_delete, template_type='email')
 
     # will be deleted as service has no custom retention, but past our default 7 days
```

Comment:
[suggestion (nonblocking)] lets delete (or update) this comment, as it's now out of date.